docs/modules/ROOT/pages/capacity-planning.adoc

```diff
@@ -585,8 +585,9 @@ of your Hazelcast data pipeline.
 
 === Upgrade Your JDK
 
-Hazelcast does not support JDK 8, so if you are still on JDK 8, please upgrade it.
-The garbage collectors of JDK 11+ have been improved to achieve much higher throughputs. 
+Hazelcast supports the **last two Java LTS versions** available at the time of its release. Newer JDK releases often contain performance improvements, consider upgrading to 
```

Comment:
```suggestion
Hazelcast supports the **current** Java LTS version and the **previous** Java LTS version, as available at the time of the Hazelcast release. Newer JDK releases often contain performance improvements, consider upgrading to 
```

Reply:
Alternate version:
```suggestion
Hazelcast supports the last two Java LTS versions (i.e. the current Java LTS version **and** the previous Java LTS version) available at the time of the Hazelcast release. Newer JDK releases often contain performance improvements, consider upgrading to 
```

Reply:
> I'm not sure it's a good idea to put our policy buried on one of these pages. If Java 23 LTS comes out tomorrow, we won't be supporting it until the next major release of Hazelcast - whereas if it says we support the latest 2 LTS releases, it would be implied that 23 was supported today.

You're right @JackPGreen.

@frant-hartm , @oliverhowell , I'd remove the first sentence in this paragraph, and stick with a phrase something like ```Newer JDK releases often contain performance improvements, so we strongly recommend you to upgrade to a newer JDK version. See the xref:deploy:versioning-compatibility.adoc#supported-java-virtual-machines[supported JDKs]. Note that the garbage collectors of the newer JDK versions have been improved to achieve much higher throughputs.```, where in the given link we list the exact supported JDK versions for that versioned documentation content.

![Screenshot 2024-02-04 at 12 01 59 PM](https://github.com/hazelcast/hz-docs/assets/5937333/614d6063-b56a-4b91-a086-d5d5ffea65d3)

Reply:
> If Java 23 LTS comes out tomorrow, we won't be supporting it until the next major release of Hazelcast

Yeah, if that's the case we need to be explicit about versions we support and agree with the general approach of linking to https://docs.hazelcast.com/hazelcast/latest/deploy/versioning-compatibility